bucket/regeditor.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "12.0.2172",
+    "description": "Freeware for editing the Windows Registry.",
```

Comment:
No need to specify license in description.

```suggestion
    "description": "Windows Registry editor.",
```

Reply:
Thanks, the description is copy from the website without change, and it is fixed now.